src/core/src/core_logic/PatchInstaller.py

```diff
@@ -342,6 +347,11 @@ def log_final_metrics(self, maintenance_window, patch_installation_successful, m
                                                         "Completed processing packages!")
         self.composite_logger.log(progress_status)
 
+        if failed_updates_for_esm_required_count > 0:
```

Comment:
failed_updates_for_esm_required_count is initialized as local variable in the method install_updates. Need to make it global variable to access here.

Reply:
Not needed anymore. Have removed it completely.